### PR TITLE
Translation of `Learn/Forms/Form_validation` page

### PR DESCRIPTION
**Pull request by josielrocha**
### Description

Translates `Learn/Forms/Form_validation` page

### Motivation

There was opened pull requests before markdown conversion (https://github.com/mdn/translated-content/pull/2960)

### CODE REVIEW

**Comment by josielrocha** [comment]
Hi @SphinxKnight!

Sorry for bothering you about this..
I just opened this Pull Request and it is not building. The "PR Test / tests (pull_request)" just keeps failling and I could not figure out what is happening. Could you please help me find what is wrong?

File: files/pt-br/learn/forms/form_validation/index.md [hunk 1]
```diff
@@ -166,42 +166,42 @@ input:valid {
...
 
-> **Nota:** Some {{HTMLElement("input")}} element types do not need a {{htmlattrxref("pattern","input")}} attribute to be validated. Specifying the `email` type for example validates the inputted value against a regular expression matching a well-formed email address (or a comma-separated list of email addresses if it has the {{htmlattrxref("multiple","input")}} attribute). As a further example, fields with the `url` type automatically require a properly-formed URL.
+> **Nota:** Alguns tipos de elemento {{HTMLElement("input")}} não precisam de um atributo {{htmlattrxref("pattern","input")}} para serem validados. Especificar o tipo `email`, por exemplo, valida o valor inserido em relação a uma expressão regular que corresponde a um endereço de e-mail bem formado (ou uma lista de endereços de e-mail separados por vírgula se tiver o {{htmlattrxref("multiple","input")} } atributo). Como outro exemplo, os campos com o tipo `url` requerem automaticamente um URL devidamente formado.
```
Comment (yin1999) [inline files/pt-br/learn/forms/form_validation/index.md:173]: may be this nit, after fixing this, I can build the document on my computer:

```suggestion
> **Nota:** Alguns tipos de elemento {{HTMLElement("input")}} não precisam de um atributo {{htmlattrxref("pattern","input")}} para serem validados. Especificar o tipo `email`, por exemplo, valida o valor inserido em relação a uma expressão regular que corresponde a um endereço de e-mail bem formado (ou uma lista de endereços de e-mail separados por vírgula se tiver o {{htmlattrxref("multiple","input")}} atributo). Como outro exemplo, os campos com o tipo `url` requerem automaticamente um URL devidamente formado.
```

Comment (SphinxKnight) [reply files/pt-br/learn/forms/form_validation/index.md:173]: yes that's the one @josielrocha! 
Thanks @yin1999 :) 

I don't know why https://github.com/mdn/translated-content/actions/runs/3238711717/jobs/5307264130#step:8:150 complains about line 136 while its on line 173 (or 167 if we remove frontmatter :/)

**Comment by SphinxKnight** [comment]
while we're at it (and if you allow me to nitpick here): 

you can remove the `translation_of` and `original_slug` key from the frontmatter (and maybe add `l10n/sourceCommit` (see https://github.com/mdn/mdn-community/discussions/129#discussioncomment-3191138 ))

File: files/pt-br/learn/forms/form_validation/index.md (hunk 1 above)
Comment (josielrocha) [reply files/pt-br/learn/forms/form_validation/index.md]: Done.
Thank you for your help!

**Comment by josielrocha** [comment]
> while we're at it (and if you allow me to nitpick here):
> 
> you can remove the `translation_of` and `original_slug` key from the frontmatter (and maybe add `l10n/sourceCommit` (see [mdn/mdn-community#129 (comment)](https://github.com/orgs/mdn/discussions/129#discussioncomment-3191138) ))
> 
> and while this PR is WIP, you can put it as a draft :) image

Sure,
Done.

Thank you again @SphinxKnight!

**Pull request merged**